grading/execute.cpp

```diff
@@ -190,13 +190,13 @@ std::set<std::string> get_compiled_executables(const nlohmann::json &whole_confi
 }
 
 
-bool local_executable (const std::string &program, const nlohmann::json &whole_config) {
+bool local_executable (const std::string &program, const nlohmann::json &whole_config, const bool isInstructor) {
   assert (program.size() > 3);
   assert (program.substr(0,2) == "./");
 
   std::set<std::string> executables = get_compiled_executables(whole_config);
 
-  if (executables.find(program.substr(2,program.size())) != executables.end()) {
+  if (executables.find(program.substr(2,program.size())) != executables.end() || isInstructor) {
```

Comment:
Is this the only reason to add the isinstructor flag? 
We should find another way to do this.

Reply:
Yes, can you suggest a different way of having the same functionality